modules/ROOT/pages/apikit-compatibility.adoc

```diff
@@ -0,0 +1,34 @@
+= APIkit Compatibility
```

Comment:
Consider referring to this section from the See Also in the index.

Reply:
Great catch! Thanks!